### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by JustArchi** [renovatebot/renovate#9820] [Suggest an Idea]
I've been using self-hosted renovate bot since around 2019 at my company as a replacement of dependabot for private gitlab hosting. It has never failed me since, with one very small **[bug](https://github.com/renovatebot/renovate/issues/5009)** that I took the effort to report, thanks again for fixing it.

With recent shutdown of dependabot native and decision of its team to **not** include one of the most useful features, automerge, renovate has replaced my previously-used dependabot native in my open-source repositories.

I'd like to thank you for your hard work, because thanks to your product I can keep focusing on what really matters, the open-source development, and not doing the hard manual labour of updating the dependencies myself or writing workarounds for irrational decisions made by others.

I'm more than sure that I'm not the only person who feels this way lately in regards to that situation, but since not everybody has the willing to write a thread like this, I'm writing this in the name of all the happy developers that are switching to renovate this very moment, including myself.

Thank you, for giving us choice 🎉

**Comment by rarkins** [comment]
Thank you for the positive feedback!

For what it's worth, I think the Dependabot automerge decision is understandable - if people shoot themselves in the foot with Renovate, they (mostly) blame themselves. If they shoot themselves in the foot with Dependabot, they blame GitHub.

And to the best of my knowledge, the team at GitHub are happy for you to have choice too! They have always been friendly and helpful any time we've needed, and one of our maintainers even works at Microsoft :)
